### PR TITLE
Group4 assignment5

### PR DESCRIPTION
**Pull request by sayloradm**

### CODE REVIEW

File: services/quaranteam/Dnd/.gitignore [hunk 1]
```diff
@@ -0,0 +1,32 @@
+HELP.md
```
Comment (sleddog) [inline services/quaranteam/Dnd/.gitignore]: what is this HELP.md file?

File: services/quaranteam/Dnd/Connections.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		c= new Context(splitted);
+		
+		
+		switch (app.toLowerCase())
+		{
+		
+		case "facebook":
+				c.changeStrat(new Facebook());
+				c.filter();
+				break;
```
Comment (sleddog) [inline services/quaranteam/Dnd/Connections.java]: odd white spacing

**Review by sleddog — COMMENTED** [review]
You have committed *.class files in this branch

File: services/quaranteam/Dnd/Connections.java (hunk 2 above)
Comment (sayloradm) [reply services/quaranteam/Dnd/Connections.java]: This is for a different project, it isn't supposed to be in here, oops. I'll take care of these files

File: services/quaranteam/Dnd/.gitignore (hunk 1 above)
Comment (sayloradm) [reply services/quaranteam/Dnd/.gitignore]: This gitignore file was a part of the springboot file that was generated and I realize it is unnecessary.

**Comment by sleddog** [comment]
Nice job

**Review by sleddog — APPROVED** [review]

**Pull request merged**